lib/email/receiver.rb

```diff
@@ -861,6 +872,42 @@ def process_forwarded_email(destination, user)
       true
     end
 
+    def forwarded_email_quote_forwarded(destination, user)
+      embedded = embedded_email_raw
+      raw = @before_embedded +
+        "\n\n[quote=\"Forwarded message\"]\n" +
```

Comment:
This should be translated.

Reply:
I wasn't so sure about keeping this in my commit, it was a "nice hack" I discovered while messing about with ways of formatting the forwarded message.

Localising to the default locale of the server shouldn't be hard (I'll add a commit doing that), but it feels like I'll be opening a whole can of worms trying to localise this locally client side. Happy to remove this and have it as an unnamed quote.

Reply:
I'd be fine with an unnamed quote. It's less burden on translators. If we find it confusing in the future we can always add it back.